### PR TITLE
Frontend authentication

### PR DESCRIPTION
**Pull request by Kris2ffer**

### CODE REVIEW

File: FrontEnd/.env-default [hunk 1]
```diff
@@ -1 +1,3 @@
 REACT_APP_INSTRUMENTATION_KEY=
+REACT_APP_SEPES_LOGIN_URL=http://localhost:5000/api/auth/token
```
Comment (Unicus-Tom) [inline FrontEnd/.env-default:2]: Backend forces HTTPS so these would just be redirected immediately. Might as well just start the request with https

File: FrontEnd/src/App.js [hunk 2]
```diff
@@ -62,27 +66,63 @@ class App extends React.Component {
...
     });
   }
 
   logout = () => {
     this.msalApp.logout();
     this.appInsights.clearAuthenticatedUserContext();
     this.appInsights.trackEvent({name: 'Logout'});
+    localStorage.removeItem(JWT_NAME);
+  }
+
```
Comment (Unicus-Tom) [inline FrontEnd/src/App.js:107]: Would love a comment here to explain what the function does. For quick reference

**Review by Unicus-Tom — APPROVED** [review]
Would have wished for more comments in the code, especially to explain what to change to allow for rapid testing. For example explain how to bypass real azure login. 
Or explain what testSepesAPI does without having to read all the code.

File: FrontEnd/src/App.js [hunk 3]
```diff
@@ -62,27 +66,63 @@ class App extends React.Component {
...
+      console.log(loginResponse.idToken.rawIdToken);
+      return fetch(process.env.REACT_APP_SEPES_LOGIN_URL, {
+        method: "post",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({"Usename": loginResponse.account.name, "idToken": loginResponse.idToken.rawIdToken, "Expiration": "later"})
+      });
+    }).then(respnse => respnse.text())
+      .then(jwt => {
+      this.setState({tokenId: jwt});
+      localStorage.setItem(JWT_NAME, jwt);
```
Comment (gjersvik) [inline FrontEnd/src/App.js:93]: You should maybe store tokens into sessionStorage if you don't need them to be long lived. As in living longer than a browsing session.

**Review by gjersvik — APPROVED** [review]
Looks good. Should maybe start thinking about writhing test for the code. And make the code more testable. The complexity of the UI will grow quickly.

**Pull request merged**